amt/locale/nl_NL/LC_MESSAGES/messages.po

```diff
@@ -424,13 +424,13 @@ msgid ""
 "filters or clearing them to see more\n"
 "            algorithm systems."
 msgstr ""
-"Er zijn geen Algoritme systemen die overeen komen met de geselecteerde "
-"filters. Pas de filters aan of verwijder ze om meer algortime systemen te"
-" zien."
+"Er zijn geen Algoritmesystemen die overeen komen met de geselecteerde "
```

Comment:
If you agree, we should also do this for the original English text:

```suggestion
"Er zijn geen algoritmesystemen die overeen komen met de geselecteerde "
```

Reply:
I think we need a copywriter to go through all of our copy for changes like this. I personally prefer the capitalized version.